internal/docs/pages/automation-glossary.js

```diff
@@ -30,6 +30,10 @@ class AutomationGlossary extends React.Component {
                   '$(\'div[data-cosmos-key="alert"] a[data-cosmos-key="alert.dismiss"\')'
                 ]
               },
+              {
+                component: 'Avatar',
+                selectors: ['$(\'div[data-cosmos-key="avatar"]\')']
+              },
```

Comment:
We should consider generating this automatically from component's code. It's going to be hard to maintain by hand